pkg/controller/rest/verifiable/operation.go

```diff
@@ -155,23 +159,14 @@ func (o *Operation) GeneratePresentation(rw http.ResponseWriter, req *http.Reque
 	rest.Execute(o.command.GeneratePresentation, rw, req.Body)
 }
 
-// GeneratePresentationByID swagger:route GET /verifiable/credential/{id}/presentation verifiable presentationByIDReq
+// GeneratePresentationByID swagger:route GET /verifiable/credential/presentationByID
```

Comment:
endpoint paths are case insensitive :)
Is it is a GET? should be POST

Reply:
should match existing endpoint path : `/verifiable/presentation/generate `

Reply:
existing endpoint for this was bit weird so i changed it to /verifiable/presentation/id/generate